packages/forms/resources/views/components/section.blade.php

```diff
@@ -4,12 +4,6 @@
         x-on:open-form-section.window="if ($event.detail.id == $el.id) isCollapsed = false"
         x-on:collapse-form-section.window="if ($event.detail.id == $el.id) isCollapsed = true"
         x-on:toggle-form-section.window="if ($event.detail.id == $el.id) isCollapsed = ! isCollapsed"
-        x-on:expand-concealing-component.window="
```

Comment:
Why remove this?

It's so that when a validation error occurs and a section is closed, it opens and gets scrolled into view

Reply:
See other PR

Reply:
It does not open on validation errors, as is now.

Reply:
Hmm you sure the browser event is not dispatched by `validate()`? That's a bug then

Reply:
This PR fixes scroll into view when you click the header + making the header clickable. 
The Section did not expand for me on validation errors so I did another PR related to that.